examples/seldon_custom_pytorch_deployment/README.md

```diff
@@ -0,0 +1,502 @@
+* a metadata store that uses the same database as the Kubeflow deployment as
+a backend
+* an GCP container registry (artifact registry)
+
+To have access to the GCP artifact store and container registry from your local workstation, the the Google Cloud CLI needs to be intialized and authorized locally as documented in
+[the official GCP documentation](https://cloud.google.com/sdk/docs/initializing).
+
+In addition to the stack components, Seldon Core must be installed in *the same*
+Kubernetes cluster as Kubeflow. The cluster must also be locally accessible
+through a Kubernetes configuration context. The reference used in this example
+is a Kubeflow and Seldon Core installation running in an GKE cluster, but any
+other type of Kubernetes cluster can be used, managed or otherwise.
+
+For more informations about configurate access to GKE locally  
+[Install kubectl and configure cluster access](https://cloud.google.com/kubernetes-engine/docs/how-to/cluster-access-for-kubectl)
+
+To view the list of contexts for kubectl, run the following command
+
+```bash
+kubectl config current-context
+```
+
+To configure GCP container registry access locally, run e.g.:
+
+```bash
+gcloud auth configure-docker
+```
+
+Extract the URL where the Seldon Core model server exposes its prediction API, e.g.:
+
+```bash
+export INGRESS_HOST=$(kubectl -n istio-system get service istio-ingressgateway \
+  -o jsonpath='{.status.loadBalancer.ingress[0].ip}')
+```
+
+Give Seldon Core access to the GCP artifact store in the configured namespace.
+For this we need first to create a service-account key and have it as local json file. First make sure that you have [SA-NAME]@[PROJECT-ID].iam.gserviceaccount.com service account created in the gcloud console that have sufficient permissions to access the bucket where model will be stored (i.e. Storage Object Admin).
+
+Generate the service-account key using gcloud.
+```bash
+gcloud iam service-accounts keys create gcloud-application-credentials.json --iam-account [SA-NAME]@[PROJECT-ID].iam.gserviceaccount.com
+```
+
+Give Seldon Core access to the GCP artifact store in the zenml-kubeflow namespace:
+
+```bash
+kubectl -n kubeflow create secret generic seldon-rclone-secret \
+    --from-literal=RCLONE_CONFIG_GS_TYPE='google cloud storage' \
+    --from-literal=RCLONE_CONFIG_GS_ANONYMOUS='false' \
+    --from-literal=RCLONE_CONFIG_GS_SERVICE_ACCOUNT_CREDENTIALS="$(cat <path-to-json-key>)"
+```
+
+The name of the created secret also needs to be passed to the `run.py` example
+launcher script via its `--secret` command line argument.
+
+NOTE: this is based on the assumption that Seldon Core is running in an GKE
+cluster that runs in the same project where container regsitry is defined otherwise
+the permission should be granted to GKE instances to access the registry. 
+More informations can be found at 
+[the official GCP documentation](https://cloud.google.com/artifact-registry/docs/integrate-gke)
```

Comment:
```suggestion
NOTE: this is based on the assumption that Seldon Core is running in a GKE
cluster that runs in the same project where the container registry is defined, otherwise
the permission should be granted to GKE instances to access the registry. 
More information can be found at 
[the official GCP documentation](https://cloud.google.com/artifact-registry/docs/integrate-gke)
```